### ISSUE TITLE
Unexpectedly high accuracy of 99 percent

### ISSUE THREAD

**Issue by the6thsense** [RetroCirce/HTS-Audio-Transformer#22]
I am getting a very high accuracy from first epoch only. I doubt if everything is alright. The accuracy on mivia dataset reach 93 percent in first epoch itself. and with pretrained model it reaches 99 percent. Please let me what is the scenario. (I do not have exposure to Pytorch Lightning)

my training outputs looks as follows:

LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name      | Type                   | Params
-----------------------------------------------------
0 | sed_model | HTSAT_Swin_Transformer | 28.6 M
-----------------------------------------------------
27.6 M    Trainable params
1.1 M     Non-trainable params
28.6 M    Total params
114.583   Total estimated model params size (MB)
Training: 0it [00:00, ?it/s]
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9361128142244022}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9509503372164316}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9634580012262416}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9735131820968731}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9872470876762722}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9832004904966278}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9729000613120785}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9894543225015328}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9917841814837522}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.992274678111588}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9931330472103004}
Validating: 0it [00:00, ?it/s]
cuda:0 {'acc': 0.9949724095646842}

**Comment by the6thsense** [comment]
Resolved it. Issue due to val_check_interval float 1.0. Closing this issue.
